Index: tests/unit/test_task_5_1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import pytest\r\nfrom task_5_1 import get_sum\r\n\r\n\r\n@pytest.mark.parametrize(\r\n    \"epsilon, my_row, expected_sum\",\r\n    [\r\n        (0.001, [1, 0.5, 0.25, 0.125, 0.0625, 0.03125], 1.96875),\r\n        (0.00001, [1, 1 / 4, 1 / 9, 1 / 16, 1 / 25, 1 / 36], 1.4913888888888889),\r\n        (0.001, [], 0),\r\n        (0.001, [1], 1),\r\n    ],\r\n)\r\ndef test_get_sum(epsilon, my_row, expected_sum):\r\n    assert abs(get_sum(epsilon, my_row) - expected_sum) < epsilon\r\n
===================================================================
diff --git a/tests/unit/test_task_5_1.py b/tests/unit/test_task_5_1.py
--- a/tests/unit/test_task_5_1.py	
+++ b/tests/unit/test_task_5_1.py	
@@ -1,5 +1,5 @@
 import pytest
-from task_5_1 import get_sum
+from venv.task_5_1 import get_sum
 
 
 @pytest.mark.parametrize(
